cpp/arcticdb/storage/python_bindings.cpp

```diff
@@ -100,6 +100,19 @@ void register_bindings(py::module& storage, py::exception<arcticdb::ArcticExcept
                                             return none.cast<py::object>();
                                        });
         })
+        .def("check_accessibility_of_primary_storage", &Library::check_accessibility_of_primary_storage)
+        ;
+
+    py::class_<CheckAccessibilityResult, std::shared_ptr<CheckAccessibilityResult>>(storage, "CheckAccessibilityResult")
+        .def_readonly("log_level", &CheckAccessibilityResult::log_level_)
```

Comment:
It's not communicating whether the check is successful. It's just meant to control the logging as the name of the variable suggests. The logging was done purely in C++, whose output is invisible in Jupyter, so I moved the actual logging work to Python.

In reality, such a quick, simple check cannot fully determine storage access, so the output is advisory/good for logging a warning. No end-user should programmatically rely on the return value.

Reply:
This part has to change please. We can't get in to passing spdlog log levels up to Python and programatically converting to Python log levels.

Reply:
The spdlog enum was already [exposed to Python](https://github.com/man-group/ArcticDB/blob/1ceb538931f11ae6d61ce50ab6983649f7b26437/cpp/arcticdb/python/python_module.cpp#L62) way before my change.

The Pybind indirection means it can be transparently replaced when the need arises. I'll alias to it from `log.hpp` per William's [suggestion](https://github.com/man-group/ArcticDB/pull/776#discussion_r1300129410).

Reply:
https://github.com/man-group/ArcticDB/pull/776#discussion_r1301404522 is a required change please